### DISCUSSION TITLE
Will there be support for multiple servers / switching between servers?

### DISCUSSION THREAD

**Discussion by snvoid** [prayag17/Blink#220] [Q&A] [answered]
I've been looking at different jellyfin clients and discovered this one. It really does look excellent. I wanted to know if there are plans to support the ability to add more than one server, and switch between the servers. Sort of the way you can in the official web/desktop client? But maybe more seamlessly?

**Comment by prayag17** [comment, marked as answer]
Yeah I have added that functionality should be available in alpha 3

**Comment by snvoid** [comment]
Awesome, can't wait to test it out. Thank you! Really excited for this client
